File: Server/src/schema/Trips/price.ts
import { Field, Float, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity()
@ObjectType()
export default class Price extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    priceID!: number;

    @Field(() => Int!)
    @Column()
    tripID!: number;

    @Field(() => Float!)
    @Column()
    price!: number;

    constructor() {
        super();

        this.priceID = 0;
        this.tripID = 0;
        this.price = 0;
    }
}

File: Assets/GraphQL/Model/Trips/Price.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class Price {
		[GqlSelection("priceID")] public int PriceID { get; private set; }
		[GqlSelection("tripID")] public int TripID { get; private set; }
		[GqlSelection("price")] public float _Price { get; private set; }
	}
}